### ISSUE TITLE
In TEST, all COE confirmed "completed" application are piling up 

### ISSUE THREAD

**Issue by ann-aot** [bcgov/SIMS#1829]

**Comment by michesmith** [comment]
@HRAGANBC @CarlyCotton  Can you please confirm this bug. Ann created it so I have no history. Is it still a bug. Do you believe this a priority for the PT release?

**Comment by HRAGANBC** [comment]
@michesmith @CarlyCotton Yes, bug. But not critical - it does not prevent Ministry user from erasing the pre-populated text and inputting new ones, so they can live with this one for now. 

I've marked it as enhancement, with release TBD

**Comment by HRAGANBC** [comment]
@michesmith @CarlyCotton Yes, bug. But not critical - it does not prevent Ministry user from erasing the pre-populated text and inputting new ones, so they can live with this one for now. 

I've marked it as enhancement, with release TBD
